scripts/entrypoint.sh

```diff
@@ -6,47 +6,48 @@ set -o pipefail
 # and customizes the entrypoint based on what the user has provided.
 # We derive variables from the environment instead of command line
 
-COMMAND="/bin/bash /build_static_site.sh"
-
-# Target is required, project has a default
 if [ -z "${SC_TARGET}" ]; then
    SC_TARGET="${GITHUB_WORKSPACE}/docs"        
 fi
-COMMAND="${COMMAND} --target ${SC_TARGET}"
-COMMAND="${COMMAND} --project ${SC_PROJECT}"
-
-if [ ! -z "${SC_PROJECT_FILE}" ]; then
-    COMMAND="${COMMAND} --project-file ${GITHUB_WORKSPACE}/${SC_PROJECT_FILE}"
-fi
-
-if [ ! -z "${SC_WEIGHTS}" ]; then
-    COMMAND="${COMMAND} --weights ${GITHUB_WORKSPACE}/${SC_WEIGHTS}"
-fi
 
 # Show the user where we are
 echo "Present working directory is:"
-echo "${PWD}"
+pwd
 ls
 
 # Clean up any previous runs (deployed in docs folder)
 # This command needs to be run relative to sourcecred respository
 # that is located at the WORKDIR /code
 rm -rf "${SC_TARGET}"
-echo "${COMMAND}"
-${COMMAND}
+(
+    set -x;
+    /build_static_site.sh \
+        --target "${SC_TARGET}" \
+        --project "${SC_PROJECT}" \
+        ${SC_PROJECT_FILE:+--project-file "${GITHUB_WORKSPACE}/${SC_PROJECT_FILE}"} \
+        ${SC_WEIGHTS:+--weights "${GITHUB_WORKSPACE}/${SC_WEIGHTS}"} \
+        ;
+)
 
-echo "Finished initial run, present working directory is ${PWD}"
+printf 'Finished initial run, present working directory is %s\n' "${PWD}"
 ls
 
 # This interacts with node sourcecred.js
 # Load it twice so we can access the scores -- it's a hack, pending real instance system
 # Note from @vsoch: these variable names aren't consistent - the project here referes to the project file.
-LOAD_COMMAND="node /code/bin/sourcecred.js load --project ${GITHUB_WORKSPACE}/${SC_PROJECT_FILE}"
+load_cmd=( node /code/bin/sourcecred.js load \
+    --project "${GITHUB_WORKSPACE}/${SC_PROJECT_FILE}" )
```

Comment:
Question - so we keep the definition of load_cmd here just to show the user but then run below in the subshell, correct? So keeping load_cmd is more of a courtesy to the user.

Reply:
Good catch; thanks! I meant to remove `load_cmd` along with `build_cmd`
in [`716df126e`][commit] but forgot. I’ll remove this so that they don’t
go out of sync.

[commit]: https://github.com/vsoch/cred-action/pull/4/commits/716df126e260680933fd36d7dd86cafd11ea69a5

Reply:
okay, great! This is what I would want to do as well - it's hugely likely they would get out of sync, and the action will show the commands run without the print.